sdk/program/src/rent.rs

```diff
@@ -82,6 +83,17 @@ impl Rent {
             ..Rent::default()
         }
     }
+
+    pub fn new_with_slots_per_epoch(slots_per_epoch: u64) -> Self {
```

Comment:
I have a feeling that it's more idiomatic to drop the `new_` in this case and just call the method `with_slots_per_epoch`?

Reply:
Sure.  CI's not too far along for that :wink:

Reply:
https://github.com/solana-labs/solana/pull/18291/commits/5e65316a184477446986347fa3a5f02994efa3e6